### PR TITLE
install & add sentry init config

### PR DESCRIPTION
**Pull request by braddf**
# Pull Request

## Description

Add Sentry and initial config incl. sampling filter function (to not overload the transaction quota immediate)

## How Has This Been Tested?

Tests pass

- [x] Yes

## Checklist:

- [x] My code follows [OCF's coding style guidelines](https://github.com/openclimatefix/nowcasting/blob/main/coding_style.md)
- [x] I have performed a self-review of my own code
- [ ] I have made corresponding changes to the documentation
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] I have checked my code and corrected any misspellings

### CODE REVIEW

File: pv_site_api/main.py [hunk 1]
```diff
@@ -41,6 +42,31 @@
 )
 logger = logging.getLogger(__name__)
 
+
+def traces_sampler(sampling_context):
+    # Examine provided context data (including parent decision, if any)
```
Comment (simlmx) [inline pv_site_api/main.py:47]: Consider using a docstring for this comment!

File: requirements.txt [hunk 2]
```diff
@@ -10,5 +10,6 @@ pytest-cov
 python-dotenv
 pandas
 pvsite-datamodel==0.1.25
+sentry-sdk
```
Comment (simlmx) [inline requirements.txt:13]: This file should have been removed when I switched to `poetry` (my bad!), I think you could remove it altogether.

File: README.md [hunk 3]
```diff
@@ -18,7 +18,7 @@ Site specific API for SBRI project
     poetry run uvicorn pv_site_api.main:app --reload
 
 
-## Runing the tests
+## Running the tests
```
Comment (simlmx) [inline README.md:21]: 👌

**Comment by devsjc** [comment]
LGTM with changes as per Simon's review!

**Review by devsjc — APPROVED** [review]

**Pull request merged**